pymemcache/serde.py

```diff
@@ -90,10 +85,7 @@ def python_memcache_deserializer(key, value, flags):
         return int(value)
 
     elif flags & FLAG_LONG:
```

Comment:
Isn't `FLAG_INTEGER` now?

`FLAG_LONG` seems dedicated to py2

Reply:
I think we want this for compatibility still based on the comment on line 31? https://github.com/linsomniac/python-memcached/blob/master/memcache.py#L135

Reply:
I yes, indeed. Thanks for the heads up.